app/views/admin/project_logs/_log_list.html.slim

```diff
@@ -1,18 +1,21 @@
-
 - if @logs.empty?
   = content_tag(:div, t('common.no_records'), class: 'alert alert-info')
 - else
   .show-actions.hidden-print
     a data-expand-all=".logs-index" href="#" = t('common.expand_all')
-    a data-hide-all=".logs-index" href="#" = t('common.hide_all')
+    - if @step && @context && @context == "calendar"
```

Comment:
If `@context` is `nil` then it won't equal `"calendar"` so you don't need to check `@context`.